products/omero/index.html

```diff
@@ -29,89 +29,74 @@ <h5 class="subheader">With our dedicated user help website and extensive technic
+                </p>
+                <a href="{{ site.baseurl }}/products/omero/import/" class="button expanded">Learn More</a>
+            </div>
+            <div class="column">
+                <a href="{{ site.baseurl }}/products/omero/organize/">
+                    <img class="thumbnail" src="http://placehold.it/300x150">
+                </a>
+                <h5 class="text-center">Organize</h5>
+                <p class="text-center">
+                    Annotate your data with Tags, Comments, Files and more.
+                    Browse or Search your images and share with collaborators.
+                    High Content Screening data is also supported.
+                </p>
+                <a href="{{ site.baseurl }}/products/omero/organize/" class="button expanded">Learn More</a>
+            </div>
+            <div class="column">
+                <a href="{{ site.baseurl }}/products/omero/view/">
+                    <img class="thumbnail" src="http://placehold.it/300x150">
+                </a>
+                <h5 class="text-center">View</h5>
+                <p class="text-center">
+                    Use OMERO.insight and OMERO.web to work with image data over the internet from anywhere. Browse thumbnails, image previews and view in full multi-dimensional viewer.
+                </p>
+                <a href="{{ site.baseurl }}/products/omero/view/" class="button expanded">Learn More</a>
+            </div>
+        </div>
+        <div class="row small-up-2 large-up-3">
+            <div class="column">
+                <a href="{{ site.baseurl }}/products/omero/analyze/">
+                    <img class="thumbnail" src="http://placehold.it/300x150">
+                </a>
+                <h5 class="text-center">Analyze</h5>
+                <p class="text-center">
+                    Use ImageJ, MATLAB, R, scripts and other tools to run analysis on data in OMERO and save results on the server.
+                </p>
+                <a href="{{ site.baseurl }}/products/omero/analyze/" class="button expanded">Learn More</a>
+            </div>
+            <div class="column">
+                <a href="{{ site.baseurl }}/products/omero/export/">
+                    <img class="thumbnail" src="http://placehold.it/300x150">
+                </a>
+                <h5 class="text-center">Export</h5>
+                <p class="text-center">
+                    Download your original image data from anywhere,
+                    export as OME-TIFF or movie formats or quickly create
+                    figures for publication.
+                </p>
+                <a href="{{ site.baseurl }}/products/omero/export/" class="button expanded">Learn More</a>
+            </div>
+            <div class="column">
+                <a href="#">
+                    <img class="thumbnail" src="http://placehold.it/300x150">
+                </a>
+                <h5 class="text-center">Publish? Utility Functions?</h5>
```

Comment:
Publish
Publish your raw data straight from the server using URLs to link directly to the web-based image viewer in a manuscript or embed images into your website.